javabuilder-authorizer/websocket_authorizer_function.rb

```diff
@@ -81,3 +74,22 @@ def get_token_info(context, sid)
 def get_region(context)
   context.invoked_function_arn.split(':')[3]
 end
+
+def error(client, status, function_name, sid)
+  puts "TOKEN VALIDATION ERROR: #{status} token_id: #{sid}"
+  metric_data = {
+    metric_name: status,
```

Comment:
potential solution for the all caps vs snake case issue: could we have a helper method that translates from status to metric name?

Reply:
Yep that could work! Do we like UpperCamelCase?

Reply:
That's how AWS's default metrics are formatted, so I like it for that consistency.